.azure-pipelines/Linux-CI.yml

```diff
@@ -94,6 +93,22 @@ jobs:
         exit 1
       fi
 
+      # onnx.checker/ORT backend test uploaded backend data
+      pip install -i https://test.pypi.org/simple/ ort-nightly
+      python workflow_scripts/test_generated_backend.py
+      if [ $? -ne 0 ]; then
+        echo "check uploaded backend data failed"
+        exit 1
+      fi
+      # onnx.checker all generated backend data
+      python onnx/backend/test/cmd_tools.py generate-data
+      # Not running ORT test because the latest opset_version may not be supported yet
+      python workflow_scripts/test_generated_backend.py --target onnx
```

Comment:
Still, running ORT test for generated backend testdata can prevent future test failures in ORT... May need to figure out a way to make ORT run unsupported opset_version. Or we can run this only before every ONNX release.